### PR TITLE
Fail fast on transfer

### PR DESCRIPTION
**Pull request by pitiphong-p**

### CODE REVIEW

File: OmiseSwift/Transfer.swift [hunk 1]
```diff
@@ -20,6 +20,7 @@ public struct Transfer: OmiseResourceObject {
     public var createdDate: Date
     
     public var status: TransferStatus
+    public let isFailFast: Bool
```
Comment (chakrit) [inline OmiseSwift/Transfer.swift]: @pitiphong-p I think this should be `shouldFailFast`

**Review by chakrit — APPROVED** [review]
Otherwise 👍

File: OmiseSwift/Transfer.swift (hunk 1 above)
Comment (pitiphong-p) [reply OmiseSwift/Transfer.swift]: Oh sounds interesting

Comment (pitiphong-p) [inline OmiseSwift/Transfer.swift]: It's the property on `Transfer` so it sounds like `isFailFast` is OK to me. But we can use the name `shouldFailFast` on the `TransferParams` instead?

**Review by igroomgrim — APPROVED** [review]
👍

**Pull request merged**